configure.ac

```
@@ -7293,7 +7293,9 @@ PY_STDLIB_MOD([_socket],
                     -a "$ac_cv_header_netinet_in_h" = "yes"]))
 
 dnl platform specific extensions
-PY_STDLIB_MOD([grp], [], [test "$ac_cv_func_getgrgid" = yes -o "$ac_cv_func_getgrgid_r" = yes])
+PY_STDLIB_MOD([grp], [],
+  [test "$ac_cv_func_getgrent" = yes &&
+   test "$ac_cv_func_getgrgid" = yes -o "$ac_cv_func_getgrgid_r" = yes])
```

Quoting the Autoconf guide for portable shell programming:

> The `-a`, `-o`, `(`, and `)` operands are not present in all implementations, and have been marked obsolete by Posix 2008. This is because there are inherent ambiguities in using them.

Let's fix the `-o` usage now that we have the opportunity:

```suggestion
  [test "$ac_cv_func_getgrent" = "yes" &&
   { test "$ac_cv_func_getgrgid" = "yes" || test "$ac_cv_func_getgrgid_r" = "yes"; }])
```

(You'll have to regen configure after applying this suggestion.)